docs/Makefile

```diff
@@ -2,13 +2,22 @@
 
 IMAGE = grafana/docs-base:latest
 
-docs:
-	docker pull ${IMAGE}
+docs-generate-frontend:
+	yarn install --pure-lockfile --no-progress
```

Comment:
Is `--no-progress` useful locally? Should think a human user would like to see progress.